### PR TITLE
Add an initial section about limits

### PR DESCRIPTION
**Pull request by certik**
I found a good intro to limits that I wrote long time ago here: https://github.com/sympy/svn_materials/blob/4c79cfc68a85a947c5d730ec57540b6dff322f2d/presentations/SD6.pdf

### CODE REVIEW

**Comment by certik** [comment]
After #11 is resolved, I'll fix the citation.

**Comment by certik** [comment]
@moorepants what's the best setup, so that the PR gets built and people can check the pdf that it looks good before merging?

**Comment by asmeurer** [comment]
Sharelatex only seems to build the pdf on master. We could build the latex on Travis (that would at least verify that it builds without errors). It would be better to use some other CI, though, as I think this will share the Travis queue with SymPy.

**Comment by certik** [comment]
Yes, it will. We should have left the repository under your name.

**Comment by asmeurer** [comment]
Maybe if we use branches on this repo, then sharelatex will build them.

File: paper.tex [hunk 1]
```diff
@@ -4,5 +4,18 @@
 \begin{document}
 \section{Introduction}
```
Comment (asmeurer) [inline paper.tex:6]: Need `\section{Features}` here.

**Comment by asmeurer** [comment]
To make it easier to collaborate on this, would it be better to split the different sections into different files? I've never real-time collaborated on a LaTeX document on GitHub before, so I don't know what the best way to do things is.

**Comment by moorepants** [comment]
The initial commits are a pain. It's usually good to have someone to start the document with sections or some base content. Then it also helps to assign people to sections for the first drafts. Then the editing with github is easier after that.

Another option is to enforce a rule where you have to put a carriage return in the tex source after each sentence. Then diffs play more nicely.

**Comment by asmeurer** [comment]
Look good? https://github.com/sympy/sympy-paper/pull/15/files

**Comment by certik** [comment]
I've rebased on top of the latest master.

**Comment by leosartaj** [comment]
We also have an algorithm based on the paper "Computing limits of Sequences" by Manuel Kauers. Maybe we can also add an eg. for this. 

```
>>> limit(Sum(1/x, (x, 1, y)) - log(y), y, oo)
EulerGamma
```

**Comment by certik** [comment]
@asmeurer would you mind reviewing this (or assigning somebody)? I put all my notes from [this](https://github.com/sympy/svn_materials/blob/4c79cfc68a85a947c5d730ec57540b6dff322f2d/presentations/SD6.pdf) old presentation. All the information is there, it just needs to be polished, so that it reads nicely. This can probably be done after it's merged into the first draft.

File: algorithms.tex [hunk 2]
```diff
@@ -15,6 +15,76 @@ \subsection{The Risch Algorithm}
 
 \subsection{The Gruntz Algorithm}
 
+The limit modules implements the Gruntz algorithm
```
Comment (thilinarmtb) [inline algorithms.tex]: `limit module` or `implement`

File: algorithms.tex [hunk 3]
```diff
@@ -15,6 +15,76 @@ \subsection{The Risch Algorithm}
...
+Out[2]: E
+\end{verbatim}
+
+\subsubsection{Details}
+
+We first define comparability classes by calculating $L$:
+\begin{equation}
+L\equiv \lim_{x\to\infty} {\log |f(x)| \over \log |g(x)|}
+\end{equation}
+And then we define the $<$, $>$ and $\sim$ operations as follows: $f>g$ when
```
Comment (asmeurer) [inline algorithms.tex:36]: Should this be `\lesssim` and `\gtrsim`?

Comment (certik) [inline algorithms.tex:36]: I don't think so: `\lesssim` is ≲,  `\gtrsim` is ≳. I want `<` and `>`.

File: algorithms.tex (hunk 2 above)
Comment (certik) [inline algorithms.tex]: Thanks, fixed in c1e7e0ec57506ce2768ecb5274657a10cbe61ced.

**Comment by moorepants** [comment]
I'm going to go ahead and merge this as it seems to be ready.

**Comment by certik** [comment]
Thanks @moorepants.

**Pull request merged**